site/en/blog/how-construct3-uses-the-file-system-access-api/index.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'How the game editor Construct 3 uses the File System Access API to let users save their games'
+subhead: >
+  The File System Access API allows read, write, and file management capabilities. Learn how Construct 3 makes use of this API.
+date: 2023-02-02
+# updated: 2023-02-02
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/p3aRD2yAMa9JUUFZRqEE.png
+alt: Four direction bomber man clone.
+authors:
+  - thomassteiner
+  - ashleygullen
+tags:
+  - capabilities
+  - case-study
+  - fugu-case-study
+---
+
```

Comment:
## Introduction

(This article is also available in form of a video.)

{% YouTube id="Y40vMQap9fs", startTime="491" %}